docs/configuration_file.md

```diff
@@ -84,12 +88,39 @@ username = <em>&lt;username&gt;</em>
 native_fiat = <em>&lt;native_fiat&gt;</em>
 </pre>
 
-Notes: 
+Notes:
 * The `username` parameter is optional and denotes the username used when connecting to the Binance.com mining pool. Only basic mining deposits (type 2) are currently supported.
 * On May 8th, 2021, Binance.com implemented a new unified dividend endpoint. This endpoint returns the time when interest affected a user's wallet. However, the endpoint used to retrieve interest payments previously returns the time when the interest was delivered (but hadn't yet affected the user's wallet). [See this post for more details](https://dev.binance.vision/t/time-difference-between-sapi-v1-asset-assetdividend-and-sapi-v1-staking-stakingrecord/12346/2).
 * Due to this discrepency, there may be duplicate locked savings/staked and flexible savings dividends around May 8th, 2021. Please review your payments around this time before processing your transactions with RP2.
-* [Currently only dust 'dribblets' of 100 or less crypto assets can be retrieved at once](https://dev.binance.vision/t/pagination-for-dustlog-asset-dividend-record-swap-history-bswap/4963). If you dust more than 100 crypto assets at one time the REST API will not be able to process the transactions successfully. 
+* [Currently only dust 'dribblets' of 100 or less crypto assets can be retrieved at once](https://dev.binance.vision/t/pagination-for-dustlog-asset-dividend-record-swap-history-bswap/4963). If you dust more than 100 crypto assets at one time the REST API will not be able to process the transactions successfully.
+<<<<<<< Updated upstream
```

Comment:
The section marked by `<<<<<<<`... `>>>>>>>` is a Git conflict and needs to be resolved manually.